src/main/java/com/github/junit5docker/ContainerInfo.java

```diff
@@ -0,0 +1,26 @@
+package com.github.junit5docker;
+
+import java.util.Collection;
+
+import static java.util.Collections.emptySet;
+import static java.util.Collections.unmodifiableCollection;
+
+class ContainerInfo {
+
+    private final String containerId;
+
+    private final Collection<String> networkIds;
+
+    ContainerInfo(String containerId, Collection<String> networkIds) {
+        this.containerId = containerId;
+        this.networkIds = networkIds == null ? emptySet() : unmodifiableCollection(networkIds);
+    }
+
+    String getContainerId() {
```

Comment:
oh, i forgot it last time :) Since this is just a immutable data structure, public fields or fine for me. I don't see any value in getters.
What's your opinion on this ?

Reply:
Hmm I've always done this out of habit after reading effective java. But on further inspection its fine to access fields directly if the class isn't public, and even more so if the fields are immutable.
So sure, make that change. Thanks! :)